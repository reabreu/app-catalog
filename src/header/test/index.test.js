import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import { TopNav } from "../index";

describe("<TopNav /> spec", () => {
  it("Renders TopNav with home url", () => {
    const { asFragment } = render(
      <Router>
        <TopNav />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
