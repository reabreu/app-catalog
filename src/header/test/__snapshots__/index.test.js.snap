// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopNav /> spec Renders TopNav with home url 1`] = `
<DocumentFragment>
  <header
    class="sc-AxjAm gdQmfm"
  >
    <a
      href="/"
    >
      <svg
        class="sc-AxirZ ktnnZK"
        height="29"
        width="160"
        xmlns="http://www.w3.org/2000/svg"
        xmlns:xlink="http://www.w3.org/1999/xlink"
      >
        <defs>
          <lineargradient
            id="b"
            x1="9.82%"
            x2="9.82%"
            y1="100%"
            y2="0%"
          >
            <stop
              offset="0%"
              stop-color="#D04358"
            />
            <stop
              offset="100%"
              stop-color="#E87F2C"
            />
          </lineargradient>
          <path
            d="M17.075 28.14h-.004a.255.255 0 01-.195-.407c.278-.37.566-.87.677-1.43a6.503 6.503 0 01-1.979.316 6.502 6.502 0 01-2.207-.383c.1.586.4 1.11.688 1.496.15.2-.048.476-.283.392-2.91-1.037-3.82-2.757-3.934-4.349a8.304 8.304 0 01-7.682-8.261 8.31 8.31 0 016.311-8.095L7.68 4.938a.453.453 0 00-.647-.046L.665 10.58c-.286.253-.662.006-.665-.295v-.007c0-.082.03-.167.094-.247L7.492.554a1.011 1.011 0 011.616.06l3.034 4.391c1.038-.57 2.26-.893 3.589-.897a6.818 6.818 0 013.378.864L22.241.481a1.011 1.011 0 011.617-.06l7.398 9.477c.307.38-.206.873-.571.55l-6.368-5.69a.453.453 0 00-.647.046l-.807 2.54a8.31 8.31 0 016.464 8.082c.014 4.52-3.57 8.2-8.049 8.332-.195 1.597-1.183 3.326-4.119 4.367a.251.251 0 01-.084.016z"
            id="a"
          />
        </defs>
        <g
          fill="none"
          fill-rule="evenodd"
        >
          <g>
            <mask
              fill="#fff"
              id="c"
            >
              <use
                xlink:href="#a"
              />
            </mask>
            <path
              d="M0 28.14h159.617V.047H0z"
              fill="url(#b)"
              mask="url(#c)"
            />
          </g>
          <path
            d="M5.14 18.8c1.706 2.832 4.748 4.09 6.794 2.81 2.046-1.281 2.321-4.615.615-7.446-1.707-2.832-4.749-4.089-6.795-2.808-2.046 1.28-2.32 4.613-.614 7.445"
            fill="#FFFFFE"
          />
          <path
            d="M7.317 19.296s-.08-.263-.08-.744.08-.722.08-.722l2.23-.94-2.23-.95s-.08-.265-.08-.734c0-.493.08-.734.08-.734l3.858 1.639s.081.275.081.756c0 .47-.08.791-.08.791l-3.86 1.638"
            fill="#D44941"
          />
          <path
            d="M26.632 18.817c-1.698 2.827-4.74 4.074-6.792 2.785-2.053-1.29-2.34-4.627-.643-7.454 1.699-2.828 4.74-4.074 6.792-2.785 2.053 1.29 2.34 4.627.643 7.454"
            fill="#FFFFFE"
          />
          <path
            d="M19.777 17.688s-.079-.22-.079-.803c0-.585.079-.805.079-.805h5.059s.089.22.089.805c0 .584-.09.803-.09.803h-5.058"
            fill="#D44941"
          />
          <path
            d="M47.14 14.4c.54 0 .652.247.607.63l-.494 4.033c-.247 1.937-1.55 3.852-5.884 3.852-3.818 0-5.188-1.645-5.188-3.424 0-.158.022-.406.045-.608l.943-7.795c.225-1.937 1.55-3.852 5.884-3.852 3.818 0 5.188 1.644 5.188 3.446 0 .136-.023.361-.045.609l-.067.609c-.045.472-.247.675-.718.675h-2.314c-.472 0-.629-.27-.561-.675l.022-.158c.023-.158.023-.27.023-.361 0-.744-.652-1.104-1.685-1.104-1.078 0-1.954.428-2.089 1.465L40 18.229a3.73 3.73 0 00-.045.451c0 .833.629 1.193 1.707 1.193 1.213 0 1.954-.428 2.066-1.463l.112-.969h-1.842c-.471 0-.65-.203-.584-.654l.225-1.756c.045-.473.27-.631.741-.631h4.762M54.02 9.77c-.071.471-.26.651-.753.651h-2.421c-.494 0-.66-.18-.589-.65l.236-1.862c.07-.471.282-.673.776-.673h2.422c.493 0 .634.202.563.673L54.02 9.77zm-1.6 12.471c-.07.472-.258.674-.752.674h-2.422c-.493 0-.658-.202-.587-.674l1.27-9.869c.046-.47.258-.672.752-.672h2.421c.494 0 .659.201.589.672l-1.27 9.87zM58.384 18.87c-.022.182-.022.272-.022.317 0 .34.112.632.787.632.855 0 1.822-.587 2.34-.926l.45-3.615c-.428-.34-1.26-.927-2.115-.927-.81 0-.922.498-.99.95l-.45 3.57zm2.857 2.803c-.698.496-1.687 1.242-3.51 1.242-1.956 0-3.058-1.084-3.058-2.87 0-.18.022-.384.045-.61l.585-4.7c.27-2.146 1.26-3.479 3.643-3.479 1.62 0 2.745.678 3.33 1.175l.09-.316c.135-.452.247-.678.72-.678h2.271c.338 0 .63.158.563.678l-1.215 9.941c-.067.475-.27.678-.742.678h-2.16c-.382 0-.562-.18-.562-.678v-.383zM71.24 11.836c.473 0 .585.177.585.642v.377c.674-.51 1.799-1.197 3.508-1.197 1.777 0 3.059 1.108 3.059 2.704 0 .066 0 .155-.045.576l-.9 7.311c-.067.466-.27.666-.741.666h-2.272c-.383 0-.653-.134-.585-.666l.787-6.425c.046-.4.046-.465.046-.51 0-.443-.18-.62-.765-.62-.878 0-1.867.554-2.339.886l-.833 6.67c-.066.465-.247.665-.719.665H67.71c-.404 0-.63-.155-.561-.666l1.214-9.771c.045-.465.247-.642.72-.642h2.158M84.302 18.494c-.023.198-.023.287-.023.375 0 .552.366.773 1.323.773h.547c.5 0 .66.265.615.662l-.205 1.568c-.069.464-.274.64-.752.64h-1.14c-2.986 0-4.125-.905-4.125-2.738 0-.22.022-.463.045-.706l.593-4.792h-1.003c-.501 0-.615-.22-.57-.64l.16-1.347c.045-.464.273-.574.752-.64l1.002-.133h.023l.25-2.01c.047-.463.274-.662.753-.662h2.348c.479 0 .593.243.547.663l-.251 2.01h1.846c.502 0 .64.242.593.617l-.182 1.502c-.046.463-.297.64-.775.64h-1.847l-.524 4.218M100.02 11.584v-.135c0-.924-.627-1.284-1.614-1.284-1.03 0-1.77.383-1.906 1.42l-.021.224v.135c0 .586.38.744.604.834l3.61 1.284c1.457.518 2.398 1.644 2.398 3.04 0 .158-.022.316-.045.564l-.156 1.397c-.225 1.937-1.66 3.852-5.919 3.852-3.833 0-5.313-1.667-5.313-3.47 0-.112 0-.247.045-.562l.157-1.33c.046-.472.269-.676.74-.676h2.332c.47 0 .604.249.538.677l-.112.833c-.023.158-.023.27-.023.406 0 .878.784 1.193 1.86 1.193 1.099 0 1.95-.383 2.107-1.419l.046-.293c.022-.157.022-.247.022-.27 0-.72-1.054-1.127-1.749-1.396l-3.07-1.217c-.897-.338-1.749-1.33-1.749-2.613 0-.135.022-.338.045-.563l.134-1.127c.224-1.937 1.457-3.852 5.65-3.852 3.765 0 5.088 1.667 5.088 3.401 0 .091 0 .18-.022.451l-.09.834c-.045.472-.27.676-.74.676h-2.332c-.425 0-.582-.158-.538-.676l.023-.338M120.76 11.256c.472 0 .787.203.585.675l-4.092 9.973c-.18.451-.27.609-.741.609h-2.766c-.472 0-.562-.203-.607-.609l-.382-4.3a22.41 22.41 0 01-.068-1.238c-.022.023-.224.72-.427 1.239l-1.82 4.3c-.18.405-.27.608-.743.608h-2.697c-.472 0-.54-.18-.607-.609l-1.461-9.973c-.068-.472.247-.675.719-.675h2.158c.472 0 .63.27.674.675l.675 4.998c.067.54.045 1.149.045 1.17 0-.044.225-.72.404-1.17l2.024-4.998c.157-.404.337-.675.81-.675h2.18c.472 0 .63.27.674.675l.45 4.998c.045.495.068 1.126.068 1.17 0-.044.202-.72.382-1.17l1.888-4.998c.157-.404.337-.675.81-.675h1.865M125.118 18.87c-.022.182-.022.272-.022.317 0 .34.113.632.787.632.855 0 1.822-.587 2.34-.926l.45-3.615c-.428-.34-1.26-.927-2.115-.927-.81 0-.922.498-.99.95l-.45 3.57zm2.857 2.803c-.697.496-1.687 1.242-3.508 1.242-1.958 0-3.06-1.084-3.06-2.87 0-.18.023-.384.045-.61l.585-4.7c.27-2.146 1.26-3.479 3.644-3.479 1.62 0 2.743.678 3.328 1.175l.09-.316c.135-.452.248-.678.72-.678h2.273c.336 0 .63.158.561.678l-1.214 9.941c-.068.475-.27.678-.742.678h-2.16c-.382 0-.562-.18-.562-.678v-.383zM142.299 12.3l-.222 1.73c-.065.464-.265.598-.752.664-1.506.177-2.614.797-3.123 1.219l-.775 6.336c-.066.466-.243.666-.709.666h-2.28c-.465 0-.62-.2-.554-.666l1.196-9.771c.066-.465.244-.642.708-.642h2.127c.465 0 .575.177.575.642l-.022.62c.62-.62 1.55-1.44 3.233-1.44.465 0 .665.178.598.643M146.411 11.836c.473 0 .563.177.563.642v.377c.652-.51 1.553-1.197 3.398-1.197 1.26 0 2.407.576 2.811 1.463.608-.443 1.53-1.463 3.758-1.463 1.755 0 3.059 1.086 3.059 2.637 0 .133 0 .222-.022.443l-.923 7.511c-.067.466-.27.666-.742.666h-2.272c-.382 0-.63-.222-.584-.666l.787-6.425c.044-.4.044-.465.044-.51 0-.443-.247-.62-.765-.62-.876 0-1.777.532-2.228.886v.045l-.81 6.624c-.067.466-.27.666-.742.666h-2.271c-.382 0-.63-.222-.585-.666l.81-6.603c.022-.221.022-.287.022-.332 0-.443-.247-.62-.766-.62-.876 0-1.755.532-2.204.886l-.831 6.67c-.068.465-.249.665-.721.665h-2.317c-.405 0-.607-.222-.562-.666l1.214-9.771c.068-.465.247-.642.721-.642h2.159"
            fill="#FFF"
          />
        </g>
      </svg>
    </a>
  </header>
</DocumentFragment>
`;
